File: python/sort/insertionsort.py
def insertion_sort(arr):
    for i in range(len(arr)):
        j = i
        while j > 0:
            if arr[j-1] > arr[j]:
                current = arr[j]
                arr[j] = arr[j-1]
                arr[j-1] = current
            j = j - 1
    return arr
 

File: go/sort/insertionsort.go
package sort

func InsertionSort(input []int64) []int64 {
	for i := 1; i < len(input); i++ {
		j := i
		for j > 0 {
			if input[j] < input[j-1] {
				current := input[j]
				input[j] = input[j-1]
				input[j-1] = current
			}
			j--
		}
	}
	return input
}
